keylime/tenant.py

```diff
@@ -543,7 +542,7 @@ def do_cvdelete(self):
             deleted = False
             for _ in range(12):
```

Comment:
Can we add a comment that this is a retry block (upon failure)?

Reply:
yep